pkg/sql/sem/builtins/pg_builtins.go

```diff
@@ -1145,6 +1157,74 @@ SELECT description
 		},
 	),
 
+	"pg_relation_is_updatable": makeBuiltin(
+		defProps(),
+		tree.Overload{
+			Types: tree.ArgTypes{{"reloid", types.Oid}, {"include_triggers", types.Bool}},
+			ReturnType: tree.FixedReturnType(types.Int4),
+			Fn: func(ctx *tree.EvalContext, args tree.Datums) (tree.Datum, error) {
+				oidArg := tree.MustBeDOid(args[0])
+				oid := int(oidArg.DInt)
+				table, err := ctx.Planner.GetImmutableTableInterfaceByID(ctx.Ctx(), oid)
+				if err != nil {
+					return zero, nil
+				}
+				tableDesc, ok := table.(catalog.TableDescriptor)
+				if !ok || !tableDesc.IsTable() || tableDesc.IsVirtualTable() {
+					return zero, nil
+				}
+
+				return allEvents, nil
+			},
+			Info: "pg_relation_is_updatable was created for compatibility. This " +
+				"should return the update events the relation supports, but as crdb " +
+				"does not support updatable views or foreign tables, right now this " +
+				"basically return allEvents or none.",
+			Volatility: tree.VolatilityStable,
+		},
+	),
+
+	"pg_column_is_updatable" : makeBuiltin(
+		defProps(),
+		tree.Overload{
+			Types: tree.ArgTypes{
+				{"reloid", types.Oid},
+				{"attnum", types.Int2},
+				{"include_triggers", types.Bool},
+			},
+			ReturnType: tree.FixedReturnType(types.Bool),
+			Fn: func(ctx *tree.EvalContext, args tree.Datums) (tree.Datum, error) {
+				oidArg := tree.MustBeDOid(args[0])
+				attNumArg := tree.MustBeDInt(args[1])
+				oid := int(oidArg.DInt)
+				attNum := uint32(attNumArg)
+				if attNumArg < 0 {
+					// system columns are not updatable
+					return tree.DBoolFalse, nil
+				}
+				table, err := ctx.Planner.GetImmutableTableInterfaceByID(ctx.Ctx(), oid)
+				if err != nil {
+					return tree.DBoolFalse, nil
+				}
+				tableDesc, ok := table.(catalog.TableDescriptor)
+				if !ok || !tableDesc.IsTable() || tableDesc.IsVirtualTable() {
+					return tree.DBoolFalse, nil
+				}
+
+				column, err := tableDesc.FindColumnWithID(descpb.ColumnID(attNum))
+				if err != nil || column.IsComputed() {
+					return tree.DBoolFalse, nil
+				}
+
+				return tree.DBoolTrue, nil
+			},
+			Info: "pg_column_is_updatable was created for compatibility. This " +
```

Comment:
I would also make this Info the simplest thing possible:

```
Info: `Returns whether the given column can be updated.`
```

and leave the not virtual stuff as a comment above.

Reply:
Done.